assets/js/protocol/protocol-navigation.js

```diff
@@ -185,16 +222,14 @@ if (typeof window.Mzp === 'undefined') { // eslint-disable-line block-scoped-var
     Navigation.bindEvents = function() {
         _navItemsLists = document.querySelectorAll('.mzp-c-navigation-items');
         if (_navItemsLists.length > 0) {
-            var navButtons = document.querySelectorAll('.mzp-c-navigation-menu-button');
-            for (var i = 0; i < navButtons.length; i++) {
-                navButtons[i].addEventListener('click', Navigation.onClick, false);
-            }
+            _navMenuButton = document.querySelector('.mzp-c-navigation-menu-button');
```

Comment:
Switching from `querySelectorAll` to `querySelector` removes the ability to have more than one nav menu on the same page. We're not currently using this ability but it's a very common pattern, WMO could be using it on pages with a nav and sub nav. (This raises the question... why aren't we).

I think we should preserve that capability.

Reply:
Thanks for bringing that up - I didn't think about that scenario. 👍🏻